spec.html

```diff
@@ -40161,6 +40161,15 @@ <h1>`Promise.any` Reject Element Functions</h1>
         </emu-clause>
       </emu-clause>
 
+      <emu-clause id="sec-promise.anysettled">
+        <h1>Promise.anySettled ( _iterable_ )</h1>
+        <emu-note>
+          <p>The *"anySettled"* property is an alias for the `Promise.race` and was added for consistency in the naming of promise combinators.</p>
+        </emu-note>
+        <p>The initial value of the *"anySettled"* property is the same function object as the initial value of the `Promise.race` property.</p>
+        <p>The value of the *"name"* property of this function is *"anySettled"*.</p>
```

Comment:
having a different name implies this is not the same function object as Promise.race, but rather it uses the same algorithm steps.

Reply:
Right - the options are, "different function objects", "same but the name is race", or "same but the name is anySettled". The latter requires changing `Promise.race.name` to be `'anySettled`'.

Reply:
"same but the name is `anySettled`" would be my preference, if we can get away with it (i.e. if it's Web-compatible). That way we can think of `race` as a legacy alias.

Reply:
I agree that would be the ideal. As a data point, no browser I’m aware of has ever gotten the name for race wrong, so es6-shim only has to correct the length, and not the name: https://github.com/paulmillr/es6-shim/blob/d4be3e6446b88208f87f27e48cb7e04aa2a70cdb/es6-shim.js#L2701 so it’s web compatible there. We’d want to check core-js too (including historical versions); if those are also safe, then I’d be quite confident overall.

Reply:
`core-js` does not check the name and the length of this method.